Mob/mob.gd

```diff
@@ -0,0 +1,11 @@
+extends CharacterBody2D
+
+@export var speed = 0
+
+func _ready():
+	Mobtarget.player
```

Comment:
What's this for?

Reply:
forgot to remove it when i was writing the code should be fixed by next commit